esphome/onju-voice-microwakeword.yaml

```diff
@@ -523,7 +528,10 @@ script:
     then:
       - media_player.volume_set:
           id: onju_out
-          volume: !lambda return clamp(id(onju_out).volume+volume, 0.0f, 1.0f);
+          volume: !lambda |-
+            float new_volume = clamp(id(onju_out).volume+volume, 0.0f, 1.0f);
```

Comment:
Restoring volume is an ESPHome concern and, although I fully understand and share your frustration, I don't think this is a welcome addition in this config or, at the very least, in this PR.

We should separate concerns so as to make it easy to track changes.